src/main/scala/sigmastate/utxo/transformers.scala

```diff
@@ -289,6 +290,30 @@ case class SigmaPropBytes(input: Value[SSigmaProp.type])
     input.cost(context) + Cost.SigmaPropBytes
 }
 
+/** Decode a SigmaProp(ProveDlog) from an ergo address */
+case class ErgoAddressToSigmaProp(input: Value[SString.type])
+  extends Transformer[SString.type, SSigmaProp.type] with NotReadyValue[SSigmaProp.type] {
+  override val opCode: OpCode = OpCodes.ErgoAddressToSigmaPropCode
+
+  override def function(intr: Interpreter, ctx: Context[_], bal: EvaluatedValue[SString.type]): Value[SSigmaProp.type] =
+    intr match {
+      case ergoInterpreter: ErgoLikeInterpreter =>
+        ErgoAddressEncoder(ergoInterpreter.networkPrefix)
```

Comment:
@ergomorphic what do you think of interpreter type check and adding `ErgoLikeInterpreter.networkPrefix`? I'm not sure it's the best way, but I wanted to express address bounds to a particular network instance as well as network type(ergo).

Reply:
@greenhat I think such kind of information belongs to meta-store. In all modern DBs there is special database called `master` which contains meta-information about main other main databases. This information is typically available for SQL queries (modulo access rights). In our case we can put it in context variables, or better into new context property of Metadata type, which is a structure with potentially many fields.